config/src/config/test_data/validator.config.yaml

```diff
@@ -0,0 +1,57 @@
+base:
+    data_dir: "/opt/libra/data/common"
+    role: "validator"
+    waypoint:
+        type: "from_storage"
+        backend:
+            type: "vault"
+            server: "https://127.0.0.1:8200"
+            ca_certificate: "/full/path/to/certificate"
+            token:
+                type: "from_disk"
```

Comment:
[For later] Ideally the user would either just set `token_file: /foo` or `token: s.foo`.

Reply:
I think this might be easier in Yaml